specification/ws/src/main/scripts/org/kaazing/specification/ws/limits/should.fail.max.lifetime.exceeded/handshake.request.and.frame.rpt

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+property location 'http://localhost:8080/echoAuth'
+property key ${ws:handshakeKey()}
+property validCredentials ${ws:base64Encode("joe:welcome")}
+property authHeader ${ws:append("Basic ", validCredentials)}
```

Comment:
Nice, I really like the way you did this, much more readable than just putting in the hex base64 encoding value!